tracer/tools/PipelineMonitor/PipelineMonitor.sln

```diff
@@ -0,0 +1,20 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PipelineMonitor", "PipelineMonitor.csproj", "{EA9CCCA0-063B-4DD0-B4F6-3F589B44CB05}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "_build", "..\..\..\build\_build.csproj", "{D477398F-D4A3-4142-BE68-C6EE2C039D22}"
```

Comment:
Do you actually need anything from `_build.csproj` 🤔? You can probably ditch this sln file I think

Reply:
Nope indeed.I tried to play with nuke for the fun of it, then realized as I just needed to do dotnet build and run, that wasn't super clever at all, even for fun.
Fixed in cb6c553be46f7c976008c6974276aa4b0bcc6601